docs/runtime/bun-apis.md

```diff
@@ -100,6 +104,26 @@ Click the link in the right column to jump to the associated documentation.
 ---
 
 - Utilities
-- [`Bun.version`](/docs/api/utils#bun-version) [`Bun.revision`](/docs/api/utils#bun-revision) [`Bun.env`](/docs/api/utils#bun-env) [`Bun.main`](/docs/api/utils#bun-main) [`Bun.sleep()`](/docs/api/utils#bun-sleep) [`Bun.sleepSync()`](/docs/api/utils#bun-sleepsync) [`Bun.which()`](/docs/api/utils#bun-which) [`Bun.peek()`](/docs/api/utils#bun-peek) [`Bun.openInEditor()`](/docs/api/utils#bun-openineditor) [`Bun.deepEquals()`](/docs/api/utils#bun-deepequals) [`Bun.escapeHTML()`](/docs/api/utils#bun-escapehtml) [`Bun.fileURLToPath()`](/docs/api/utils#bun-fileurltopath) [`Bun.pathToFileURL()`](/docs/api/utils#bun-pathtofileurl) [`Bun.gzipSync()`](/docs/api/utils#bun-gzipsync) [`Bun.gunzipSync()`](/docs/api/utils#bun-gunzipsync) [`Bun.deflateSync()`](/docs/api/utils#bun-deflatesync) [`Bun.inflateSync()`](/docs/api/utils#bun-inflatesync) [`Bun.inspect()`](/docs/api/utils#bun-inspect) [`Bun.nanoseconds()`](/docs/api/utils#bun-nanoseconds) [`Bun.readableStreamTo*()`](/docs/api/utils#bun-readablestreamto) [`Bun.resolveSync()`](/docs/api/utils#bun-resolvesync)
+- [`Bun.version`](/docs/api/utils#bun-version)
+  [`Bun.revision`](/docs/api/utils#bun-revision)
+  [`Bun.env`](/docs/api/utils#bun-env)
+  [`Bun.main`](/docs/api/utils#bun-main)
+  [`Bun.sleep()`](/docs/api/utils#bun-sleep)
+  [`Bun.sleepSync()`](/docs/api/utils#bun-sleepsync)
+  [`Bun.which()`](/docs/api/utils#bun-which)
+  [`Bun.peek()`](/docs/api/utils#bun-peek)
+  [`Bun.openInEditor()`](/docs/api/utils#bun-openineditor)
+  [`Bun.deepEquals()`](/docs/api/utils#bun-deepequals)
+  [`Bun.escapeHTML()`](/docs/api/utils#bun-escapehtml)
+  [`Bun.fileURLToPath()`](/docs/api/utils#bun-fileurltopath)
+  [`Bun.pathToFileURL()`](/docs/api/utils#bun-pathtofileurl)
+  [`Bun.gzipSync()`](/docs/api/utils#bun-gzipsync)
+  [`Bun.gunzipSync()`](/docs/api/utils#bun-gunzipsync)
+  [`Bun.deflateSync()`](/docs/api/utils#bun-deflatesync)
+  [`Bun.inflateSync()`](/docs/api/utils#bun-inflatesync)
+  [`Bun.inspect()`](/docs/api/utils#bun-inspect)
+  [`Bun.nanoseconds()`](/docs/api/utils#bun-nanoseconds)
+  [`Bun.readableStreamTo*()`](/docs/api/utils#bun-readablestreamto)
+  [`Bun.resolveSync()`](/docs/api/utils#bun-resolvesync)
```

Comment:
☝️ Is this auto-generated or hand maintained?

Reply:
hand maintained